Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -23,82 +28,141 @@
     understands:
     - Xeno
     - TauCetiBasic
-#
-#- type: entity
-#  name: Neutral Praetorian
-#  id: MobXenoNeutralPraetorian
-#  parent: MobXenoPraetorianNPC
-#  description: They mostly come at night. Mostly.
-#  components:
-#  - type: NpcFactionMember
-#    factions:
-#    - Passive
-#  - type: PointLight
-#    radius: 2
-#    energy: 1
-#    color: "#62B85E"
-#  - type: GhostRole
-#    name: ghost-role-information-friendlyxeno-name
-#    description: ghost-role-information-friendlyxeno-description
-#    rules: ghost-role-information-friendlyxeno-rules
-#  - type: GhostTakeoverAvailable
-#  - type: LanguageKnowledge
-#    speaks:
-#    - Xeno
-#    understands:
-#    - Xeno
-#    - TauCetiBasic
-#
-#- type: entity
-#  name: Neutral Drone
-#  id: MobXenoNeutralDrone
-#  parent: MobXenoDroneNPC
-#  description: They mostly come at night. Mostly.
-#  components:
-#  - type: NpcFactionMember
-#    factions:
-#    - Passive
-#  - type: PointLight
-#    radius: 2
-#    energy: 1
-#    color: "#8B5EB8"
-#  - type: GhostRole
-#    name: ghost-role-information-friendlyxeno-name
-#    description: ghost-role-information-friendlyxeno-description
-#    rules: ghost-role-information-friendlyxeno-rules
-#  - type: GhostTakeoverAvailable
-#  - type: LanguageKnowledge
-#    speaks:
-#    - Xeno
-#    understands:
-#    - Xeno
-#    - TauCetiBasic
-#
-#- type: entity
-#  name: Neutral Ravager
-#  id: MobXenoNeutralRavager
-#  parent: MobXenoRavagerNPC
-#  description: They mostly come at night. Mostly.
-#  components:
-#  - type: NpcFactionMember
-#    factions:
-#    - Passive
-#  - type: PointLight
-#    radius: 2
-#    energy: 1
-#    color: "#E3954D"
-#  - type: GhostRole
-#    name: ghost-role-information-friendlyxeno-name
-#    description: ghost-role-information-friendlyxeno-description
-#    rules: ghost-role-information-friendlyxeno-rules
-#  - type: GhostTakeoverAvailable
-#  - type: LanguageKnowledge
-#    speaks:
-#    - Xeno
-#    understands:
-#    - Xeno
-#    - TauCetiBasic
-#
+
```

Comment:
These neutral xenos should also preferably share a common parent like MobXenoNeutralBase. Again, you can use multiple inheritance.